nest-module/Dockerfile

```
@@ -62,21 +53,26 @@ RUN cmake \
     /cpprestsdk
 RUN ninja && ninja install
 
-#
+
 # Install NEST Simulator
-ARG NEST_VERSION=3.2
-ARG NEST_SRC_PATH=/tmp
+ARG NEST_SRC_PATH=/tmp/nest-simulator
+
+# Install dependencies for NEST Server
+RUN python3 -m pip install \
+    gunicorn \
+    Flask \
+    Flask-cors \
+    RestrictedPython
 
-# Get released files.
-RUN wget https://github.com/nest/nest-simulator/archive/refs/tags/v${NEST_VERSION}.tar.gz -P ${NEST_SRC_PATH} && \
-    cd ${NEST_SRC_PATH} && tar -zxf v${NEST_VERSION}.tar.gz
+# Clone NEST Simulator
+RUN git clone https://github.com/nest/nest-simulator.git ${NEST_SRC_PATH}
```

```suggestion
RUN git clone -depth 1 --branch v3.3 https://github.com/nest/nest-simulator.git ${NEST_SRC_PATH}
```

we can already specify the branch here and clone with depth 1 which will be faster and smaller.